gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,6 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-6.9-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-8.0-bin.zip
```

Comment:
Could you add `distributionSha256Sum`? E.g. with (for Fish shell):

```
GRADLE_VERSION=8.0 gw wrapper --gradle-version $GRADLE_VERSION --gradle-distribution-sha256-sum (wget -O- -q "https://services.gradle.org/distributions/gradle-$GRADLE_VERSION-bin.zip.sha256")
```

And "vote" for the built-in task in Gradle: https://github.com/gradle/gradle/issues/12785 ;-)

Reply:
Doing. Is this a secondary check in addition to "validate wrapper"?

Reply:
It's mostly for the local execution. To detect corrupted binaries (or some kind of MiTM attacks). The "validate wrapper" action is even better, as it (AFAIR) compares the actual checksum with the record published by Gradle.

Reply:
It is to validate the checksum on developer's machines.

Frankly speaking, I thought of asking the checksum, however, it was not there before, so I skipped

Reply:
I'm on Windows; I downloaded it manually and copy-pasted from
https\://services.gradle.org/distributions/gradle-8.0-bin.zip.sha256

https://github.com/gradle-nexus/publish-plugin/pull/168/commits/2cecfc0b193f1fe6cd362db6cdfc4dff19448526

Reply:
Alternatively, they are on: https://gradle.org/release-checksums/